README.md

```diff
@@ -920,9 +952,9 @@ random([1, 2, 3]); // one of [1, 2, 3], at random
 ```js
 import shuffle from "just-shuffle";
 
-shuffle([1, 2, 3]); 
+shuffle([1, 2, 3]);
 // array with original elements randomly sorted
-shuffle([1, 2, 3], {shuffleAll: true}); 
+shuffle([1, 2, 3], {shuffleAll: true});
```

Comment:
@angus-c there was some extraneous whitespace, which vscode autofixed. Do you want me to fix the commit, not to include it?

Reply:
that would be great thanks!